src/MinkowskiSum.jl

```diff
@@ -77,6 +77,10 @@ Minkowski sum.
 """
 +(∅::EmptySet, X::LazySet) = EmptySet()
```

Comment:
Return `∅` instead of creating a new instance.

Reply:
There are four occurrences of `EmptySet()` in this file and eight occurrences in `CartesianProduct.jl`.

Reply:
In fact the Julia compiler seems to unify all instances of this type automatically, but it is still much faster to use `∅`:
```julia
julia> @time EmptySet();
  0.000056 seconds (4 allocations: 160 bytes)
julia> @time ∅;
  0.000003 seconds (4 allocations: 160 bytes)
```

Reply:
I am beginning to wonder if `∅` is really a constant.
```julia
julia> expand(∅)
LazySets.EmptySet()
```

Reply:
`EmptySet()` always returns the same instance as well.
```julia
object_id(EmptySet())
0x34e36c91d4a34027
object_id(EmptySet())
0x34e36c91d4a34027
```
Then I do not understand why `∅` is faster...